src/Modera/SecurityBundle/PasswordStrength/StrongPasswordValidator.php

```diff
@@ -66,13 +66,39 @@ public function validate($value, Constraint $constraint)
             $lengthValidator->validate($value, $regexConstr);
         }
 
-        if ($this->config->isCapitalLetterRequired()) {
-            $errorMsg = 'Password must contain at least one capital letter.';
-            if (class_exists('Modera\FoundationBundle\Translation\T')) {
-                $errorMsg = T::trans($errorMsg);
+        if ($this->config->isLetterRequired()) {
+            switch ($this->config->getLetterRequiredType()) {
+                case 'capital_or_non_capital':
+                    $pattern = '/[A-Za-z]/';
+                    $errorMsg = 'Password must contain at least one letter.';
+                    if (class_exists('Modera\FoundationBundle\Translation\T')) {
```

Comment:
Perform extract method refactoring, it is copy-paste in 4 places here of the pretty much identical block.

Reply:
->trans('a message');

private function trans($message)
{
    return class_exists('Modera\FoundationBundle\Translation\T') ? T::trans($message) : $message;
}

Reply:
we cannot do this, because then we cannot import them